### PR TITLE
feat: add riscv64 support

### PR DESCRIPTION
**Pull request by ernado**
Should be marked as [Tier 3](https://etcd.io/docs/v3.5/op-guide/supported-platform/#support-tiers).

### CODE REVIEW

File: Dockerfile-release.riscv64 [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM --platform=linux/ppc64le riscv64/debian:sid-slim
```
Comment (serathius) [inline Dockerfile-release.riscv64:1]: New releases were migrated to gcr.io/distroless/static-debian11

Don't think we want to have use different images.

**Comment by ernado** [comment]
Looks like it would be currently hard to add riscv64 to distroless, please see https://github.com/GoogleContainerTools/distroless/pull/1268.

Can we use `riscv64/debian:sid-slim` for now?

File: Dockerfile-release.riscv64 (hunk 1 above)
Comment (ahrtr) [inline Dockerfile-release.riscv64:1]: copy paste typo?  `--platform=linux/riscv64`

Comment (ahrtr) [reply Dockerfile-release.riscv64:1]: Previously I saws lots of CVEs, so eventually we migrated to distroless base image.

Comment (ernado) [reply Dockerfile-release.riscv64]: y, fixed

Comment (serathius) [reply Dockerfile-release.riscv64]: We don't want to use busybox due to licensing issues https://github.com/etcd-io/etcd/issues/15034

**Comment by serathius** [comment]
Need to agree with distroless maintainer arches to the build is quite expensive and makes it harder to maintain. Especially if we would need to use separate base image for riscv64. For now I'm against adding support riscv64 until https://www.debian.org/ports/ and distroless supports it

File: Dockerfile-release.riscv64 (hunk 1 above)
Comment (ernado) [reply Dockerfile-release.riscv64]: It looks like `FROM scratch` just works.

**Comment by ernado** [comment]
Closing this until distroless dependency is resolved.

Will be maintaining forks and binaries under https://github.com/go-riscv.

Also, there is signed distroless image from go-riscv:
https://github.com/go-riscv/distroless

```bash
docker pull ghcr.io/go-riscv/distroless/static-unstable:latest
```

```dockerfile
FROM ghcr.io/go-riscv/distroless/static-unstable:latest
```

**Pull request closed without merging**